Dockerfile

```diff
@@ -0,0 +1,121 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Copyright 2020 (C) Olliver Schinagl <oliver@schinagl.nl>
+# Copyright 2021 (C) Cisco Systems, Inc. and/or its affiliates. All rights reserved.
+
+# hadolint ignore=DL3007  latest is the latest stable for alpine
+FROM registry.hub.docker.com/library/alpine:latest AS builder
+
+LABEL maintainer="Cisco Systems, Inc. <clamav@cisco.com>"
+
+EXPOSE 3310
+EXPOSE 7357
+
+WORKDIR /src
+
+COPY . /src/
+
+# hadolint ignore=DL3008  We want the latest stable versions
+RUN apk add --no-cache \
+        bsd-compat-headers \
+        bzip2-dev \
+        check-dev \
+        cmake \
+        curl-dev \
+        file \
+        fts-dev \
+        g++ \
+        git \
+        json-c-dev \
+        libmilter-dev \
+        libtool \
+        libxml2-dev \
+        linux-headers \
+        make \
+        ncurses-dev \
+        openssl-dev \
+        pcre2-dev \
+        py3-pytest \
+        zlib-dev \
+    && \
+    addgroup -S "clamav" && \
+    adduser -D -G "clamav" -h "/var/lib/clamav" -s "/bin/false" -S "clamav" && \
+    install -d -m 755 -g "clamav" -o "clamav" "/var/log/clamav" && \
+    mkdir -p "./build" && \
+    cd "./build" && \
+    chown "clamav:clamav" "." && \
+    su -s "/bin/sh" "clamav" -c "cmake .. \
+          -DCMAKE_BUILD_TYPE=\"Release\" \
+          -DCMAKE_INSTALL_PREFIX=\"/usr\" \
+          -DCMAKE_INSTALL_LIBDIR=\"/usr/lib\" \
```

Comment:
As mentioned before, without this, it builds in lib64. I initially had the FULL variant there, which just gets ignored/is not relevant. So maybe the same workaround that you put in the CMakeList.txt for the `CMAKE_INSTALL_FULL_LIBDIR` might be needed here? I'm not sure however.